### ISSUE TITLE
Structured list isn't resizable

### ISSUE THREAD

**Issue by werdnanoslen** [IBM/carbon-design-kit#99]
The  symbol is not resizable, so if I want fewer rows then I can't make the overall symbol the actual size of its contents.

In the latest Sketch Library (from Cloud), the `Theme: white / structured list / group / 2 column offset` symbol isn't resizable which means it can't be used at different heights without detaching it from the symbol.

![Apr-29-2019 09-13-00](https://user-images.githubusercontent.com/764090/56898618-7a6e5800-6a5f-11e9-85f7-2dffb79fac14.gif)

**Comment by aagonzales** [comment]
@werdnanoslen thats the minimum height for each row, you're not supposed to be able to shrink it but it should be able to grown larger. Is it still a problem if you're growing and not shrinking the list?

**Comment by werdnanoslen** [comment]
Oh that's fine, I meant that in my case I only needed three rows, so when I used `none` as the override for row 4, there was still white space there. And to keep my margins easy to track, I wanted to pull the bottom up to the third row, but that cause the whole thing to resize. 

This is what I wanted and why I needed to decouple from the symbol (see the bottom of the bounding box is flush with the bottom line):
![image](https://user-images.githubusercontent.com/764090/57094302-3f2d8c80-6cde-11e9-9369-8d874f5811e6.png)

And this is what I get with the current symbol (bounding box still flush with removed fourth row):
![image](https://user-images.githubusercontent.com/764090/57094444-8b78cc80-6cde-11e9-8cb6-9dc4e7bb405e.png)

So I think for each row, I'd just suggest giving it these resize rules:
![image](https://user-images.githubusercontent.com/764090/57094530-c24ee280-6cde-11e9-8078-a10e5fe129a4.png)

**Comment by werdnanoslen** [comment]
btw that broken left column header reminded me to make an issue for that: https://github.com/carbon-design-system/carbon-design-kit/issues/108

**Comment by aagonzales** [comment]
Hmm fixing the size keeps the pieces in place but it doesn't allow you to make the whole size bigger. The group version of accordion is a fickle guy.

**Comment by werdnanoslen** [comment]
Right, I don't want to make the size/spacing bigger. I just want the bounding box to fit the contents so I can keep elements' margins clear.
